### DISCUSSION TITLE
Embed the Same Figure of Computation Output in Different Slides (Reveal JS)

### DISCUSSION THREAD

**Discussion by RoyiAvital** [quarto-dev/quarto-cli#3760] [Q&A] [answered]
I have a computation which its output is a figure.
I use it on a certain slide and I want to appear in other slides as well.

At the moment, I _copy & paste_ the code into the other slides.  
I was wondering if there's a way to embed the result of computation in different slides without explicitly running the code again.

Could it be done or is it a feature to ask for?

I know there is an option to save the output to an image explicitly and then just embed the image, but I wonder if it can be done on Quarto level.

**Comment by mcanouil** [comment, marked as answer]
Late answer, but from the R point of view, see <https://bookdown.org/yihui/rmarkdown-cookbook/reuse-chunks.html>

**Reply by RoyiAvital** [reply]
Will it work on Quarto as well?

**Reply by mcanouil** [reply]
Yes because it's `knitr`.

**Reply by RoyiAvital** [reply]
OK. So maybe it should be added to Quarto's documentation as well?

**Reply by mcanouil** [reply]
Quarto can't be exhaustive of what's already described in knitr/rmarkdown and every Jupyter kernel.
For instance, nearly everything from Pandoc can be done in Quarto, but Quarto won't embed the entire Pandoc documentation.

If you are using Jupyter kernel, see for this Jupyter kernel specific documentation to know what's possible.
If you are using knitr engine, see for knitr/rmarkdown specific documentation to know what's possible.
